src/waveform/widgets/qopengl/waveformwidget.cpp

```diff
@@ -32,23 +35,23 @@ void WaveformWidget::renderGL() {
             m_rendererStack[0]->qopenglWaveformRenderer()->renderGL();
         }
     } else {
-        for (int i = 0; i < m_rendererStack.size(); ++i) {
-            m_rendererStack[i]->qopenglWaveformRenderer()->renderGL();
+        for (auto renderer : m_rendererStack) {
+            renderer->qopenglWaveformRenderer()->renderGL();
         }
     }
 }
 
 void WaveformWidget::initializeGL() {
-    for (int i = 0; i < m_rendererStack.size(); ++i) {
-        m_rendererStack[i]->qopenglWaveformRenderer()->initializeOpenGLFunctions();
-        m_rendererStack[i]->qopenglWaveformRenderer()->initializeGL();
+    for (auto renderer : m_rendererStack) {
```

Comment:
to fix clazy, same below.
```suggestion
    for (auto renderer : std::as_const(m_rendererStack)) {
```

Reply:
thanks. DONE.